app/public/javascripts/passValid.js

```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
function ValidatePassword(){
```